pkg/artifactory/resource/security/resource_artifactory_permission_target_test.go

```diff
@@ -159,15 +159,12 @@ const testLength = `
 	  repo {
 		includes_pattern = ["foo/**"]
 		repositories     = ["{{ .repo_name }}"]
-
-		actions {
-		}
```

Comment:
`actions` is an optional field in the API so I don't know why test for empty attribute. Can be framework related.